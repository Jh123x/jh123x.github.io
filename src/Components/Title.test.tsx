import * as React from "react";
import { describe, expect, test } from "@jest/globals";
import Title from "./Title";

describe("Title", () => {
  test("should render correctly", () => {
    expect(<Title title="Test Title" span="Test Span" />).toMatchSnapshot();
  });
});
