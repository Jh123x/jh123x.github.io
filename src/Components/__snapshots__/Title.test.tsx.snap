// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render correctly 1`] = `
<Title
  span="Test Span"
  title="Test Title"
/>
`;
